caql/Metrics.g4

```diff
@@ -0,0 +1,18 @@
+grammar Metrics;
+@hearder{

```

Comment:
The prefered manner to declare a package in the generated code is with `-package` on the Antlr Tool .jar generation. https://github.com/antlr/antlr4/blob/master/doc/tool-options.md

Reply:
removed this header.